kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html

```diff
@@ -19,31 +19,47 @@
         </button>
 
         <a class="node__name flex-spacer"
-           ui:sref="search.results({query: (node.data.name)})">
+           ui:sref="search.results({query: '~'+node.data.content.pathId})">
```

Comment:
[Added it to query-syntax](https://github.com/guardian/grid/commit/1cf1b463a9a69213c55b870dd62b16137d06a737)